docs/content/mongocxx-v3/installation/advanced.md

```diff
@@ -114,4 +114,209 @@ Pass `-DENABLE_TESTS=OFF` as a cmake option to disable configuration of test tar
 cmake .. -DENABLE_TESTS=OFF
 cmake --build .. --target help
 # No test targets are configured.
-```
\ No newline at end of file
+```
+
+## Installing to non-standard directories
+
+To install the C++ driver to a non-standard directory, specify `CMAKE_INSTALL_PREFIX` to the desired
+install path:
+
+```sh
+cmake ..                                            \
+    -DCMAKE_BUILD_TYPE=Release                      \
+    -DCMAKE_INSTALL_PREFIX=$HOME/mongo-cxx-driver
+```
+
+Consider also specifying the `-DCMAKE_INSTALL_RPATH=` option to the `lib` directory of the install.
+This may enable libmongocxx.so to locate libbsoncxx.so:
+
+```sh
+cmake ..                                             \
+    -DCMAKE_BUILD_TYPE=Release                       \
+    -DCMAKE_INSTALL_PREFIX=$HOME/mongo-cxx-driver    \
+    -DCMAKE_INSTALL_RPATH=$HOME/mongo-cxx-driver/lib
+```
+
+If the C driver is installed to a non-standard directory, specify `CMAKE_PREFIX_PATH` to the install
+path of the C driver:
+
+```sh
+cmake ..                                            \
+    -DCMAKE_BUILD_TYPE=Release                      \
+    -DCMAKE_PREFIX_PATH=$HOME/mongo-c-driver        \
+    -DCMAKE_INSTALL_PREFIX=$HOME/mongo-cxx-driver
+```
+
+> *Note* If you need multiple paths in a CMake PATH variable, separate them with a semicolon like
+> this:
+> `-DCMAKE_PREFIX_PATH="/your/cdriver/prefix;/some/other/path"`
+
+### Configuring with `mongocxx` 3.1.x or 3.0.x
+
+Instead of the `-DCMAKE_PREFIX_PATH` option, users must specify the `libmongoc` installation
+directory by using the `-DLIBMONGOC_DIR` and `-DLIBBSON_DIR` options:
+
+```sh
+cmake ..                                            \
+    -DCMAKE_BUILD_TYPE=Release                      \
+    -DLIBMONGOC_DIR=$HOME/mongo-c-driver            \
+    -DLIBBSON_DIR=$HOME/mongo-c-driver              \
```

Comment:
Double-checking if this is correct. Locally, I've had to specify `-Dlibbson-1.0_DIR` and `-Dlibmongoc-1.0_DIR` to point to custom install locations for the C Driver.